src/6model/containers.c

```diff
@@ -469,8 +469,8 @@ static void native_ref_fetch_i(MVMThreadContext *tc, MVMObject *cont, MVMRegiste
 
 static void native_ref_fetch_u(MVMThreadContext *tc, MVMObject *cont, MVMRegister *res) {
     MVMNativeRefREPRData *repr_data = (MVMNativeRefREPRData *)STABLE(cont)->REPR_data;
-    if (repr_data->primitive_type != MVM_STORAGE_SPEC_BP_INT)
-        MVM_exception_throw_adhoc(tc, "This container does not reference a native integer");
+    if (repr_data->primitive_type != MVM_STORAGE_SPEC_BP_UINT64)
```

Comment:
Just a general question (i.e., not specifically related to this line), but why is it `MVM_STORAGE_SPEC_BP_UINT64` instead of `MVM_STORAGE_SPEC_BP_UINT`?

Reply:
I actually don't know. I think the better question is actually "why BP_INT and not BP_INT64"? I guess those constants have grown historically. There are also constants for other sizes both for unsigned and also for signed integers. BP_INT is the only of these constants without an explicit size.

Reply:
Yeah, maybe we should change all the `MVM_STORAGE_SPEC_BP_INT` (i.e., the ones without a size) to `MVM_STORAGE_SPEC_BP_INT64` after this is merged.